tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -117,18 +117,21 @@ SELECT current_user;
 ALTER SCHEMA rename_schema RENAME TO new_name_schema;
 SET ROLE :ROLE_DEFAULT_PERM_USER;
 SELECT user_view_schema, user_view_name, partial_view_schema, partial_view_name
+       direct_view_schema, direct_view_name
       FROM _timescaledb_catalog.continuous_agg;
- user_view_schema | user_view_name | partial_view_schema | partial_view_name 
-------------------+----------------+---------------------+-------------------
- new_name_schema  | rename_test    | public              | _partial_view_3
+ user_view_schema | user_view_name | partial_view_schema | direct_view_schema | direct_view_name 
+------------------+----------------+---------------------+--------------------+------------------
```

Comment:
Comma is missing in the query and as result the column has disappeared. Very strange that there is no syntactic error on the query.

Reply:
It just rename that column (the `AS` is implicit). Fixed.